static_babel_content.json

```diff
@@ -66,7 +66,7 @@
         "DO": "Rep\u00fablica Dominicana",
         "DZ": "\u0627\u0644\u062c\u0632\u0627\u0626\u0631",
         "EC": "Ecuador",
-        "EE": "Eesti",
+        "EE": "Estonia",
```

Comment:
Maybe this was an unintended change?

Reply:
Not sure how that changed took place, maybe due to the extract function? But will undo it, thanks:))